sig-docs/README.md

```diff
@@ -99,9 +99,15 @@ The following [subprojects][subproject-definition] are owned by sig-docs:
 ### reference-docs
 - **Owners:**
   - [kubernetes-sigs/reference-docs](https://github.com/kubernetes-sigs/reference-docs/blob/master/OWNERS)
+### subproject-definition
+- **Owners:**
+  - [kubernetes/community/governance.md#subprojects](https://github.com/kubernetes/community/blob/master/governance.md#subprojects)
```

Comment:
This area is a list of SIG Docs subprojects: blog, localization, reference-docs
I don't think the subproject and WG definition is needed here.

I think this place is better for the link to the subproject definition if it's not there
https://github.com/kubernetes/community/blob/master/governance.md#subprojects

And this place is better for the link to the WG definition if it's not there
https://github.com/kubernetes/community/blob/master/governance.md#working-groups